### ISSUE TITLE
[Improvement] Use CSS variables to support easily changing colors

### ISSUE THREAD

**Issue by staghouse** [leezng/vue-json-pretty#233]
It would be nice if we could override the colors of certain elements using CSS variables instead of writing a selector and desirable color.

**Comment by leezng** [comment]
Yes. I think it's a nice idea, if you have time maybe you can help with it.
